scheduler/rankers/boefje.py

```diff
@@ -31,7 +31,8 @@ def rank(self, obj: Any) -> int:
         Since we want to have a lower bound of a priority of 3, we will use
         an exponential decay function in decreasing form.
         """
-        if obj.last_run_boefje is None:
+        # New tasks that have not yet run before
+        if obj.prior_tasks is None or len(obj.prior_tasks) == 0:
```

Comment:
if `obj.prior_tasks` is a list:
```suggestion
        if obj.prior_tasks is None or not obj.prior_tasks:
```
Otherwise no remark :)

Reply:
Weird I specifically remember that I had to check the length in some instances. Tried it in the interpreter and you're right. Have to look where I got that from.